.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+
+    - name: Publish collector locally
+      run: |
+        git clone https://github.com/snowplow/stream-collector.git
```

Comment:
Just for fun, how about.....
```
git clone --branch 2.1.0 --depth 1 https://github.com/snowplow/stream-collector.git
```
(untested)

FYI I think @peel has it on his radar to publish collector core as an asset, so this local publish might become unnecessary in future.

Reply:
Nice, would be good to have this as a published asset as it would simplify contributor set up. Took me a minute to figure out why the build was failing when I cloned `snowplow-micro` initially.